lib/components/form/user-settings.js

```diff
@@ -329,20 +329,23 @@ const TripRequest = ({
   tripRequest,
   user
 }) => {
-  const { canDelete = true, id, query, timestamp } = tripRequest
+  const { canDelete = true, query, timestamp } = tripRequest
 
   const _onSelect = useCallback(
     // Update query params and initiate search.
     () => {
-      setQueryParam(query, id)
+      // In previous versions of this code, tripRequest.id was passed to setQueryParam,
+      // however that causes an extra entry to be added to the recent searches on click,
+      // in addition to the entry added when clicking the "Plan" (magnifying glass) button.
```

Comment:
Could this be simplified to just "exclude tripRequest.id in order to avoid creating duplicate recent searches in the sidebar"?